toolkit/tools/internal/grub/filescanner/filescanner.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package filescanner
```

Comment:
If this filescanner functionality is specific to grub files, I would put this under the `package grub`. `filescanner` sounds like `fileutils` to me (usable across the board, not specific to grub files).
Please disregard if the implementation is not specific to grub usecase. I wasn't sure because you have placed it under grub/.

Reply:
It's actually a fairly generic library. I've moved the package up a directory.